README.md

```diff
@@ -1,2 +1,180 @@
-# AutoAI-code-generation
-code pattern to show notebook generation feature of AutoAI
+
+<!-- Add a new Title and fill in the blanks -->
+# Generate Python notebook for pipeline models using AutoAI
+
+## Summary
+
+In this code pattern, we will learn how to automatically generate a jupyter notebook model through AutoAI. We will explore, modify and retrain this model pipeline using python code. Lastly, we will deploy this model in Watson Machine Learning using WML APIs.
+
+## Description
+
+AutoAI is a graphical tool available within Watson Studio that analyzes your dataset, generates several model pipelines  and ranks them based on the metric chosen for the problem. This code pattern shows extended features of AutoAI. More basic AutoAI exploration for the same dataset is covered in the [Generate machine learning model pipelines to choose the best model for your problem](https://developer.ibm.com/tutorials/generate-machine-learning-model-pipelines-to-choose-the-best-model-for-your-problem-autoai/) article.
+
+When you have completed this code pattern, you will understand how to:
+
+* Run an AutoAI experiment.
+* Generate and save a Python notebook.
+* Execute notebook and analyse results.
+* Improvise pipeline model and retrain using auto-generated notebook.
+* Deploy model using Watson Machine Learning from within notebook .
+
+## Architecture components
+<!--add an image in this path-->
+![architecture](doc/source/images/architecture.png)
+
+<!--Optionally, add flow steps based on the architecture diagram-->
+## Flow
+
+1. The user submits an AutoAI experiment using default settings.
+1. Multiple pipeline models are generated. A pipeline model of choice from the leaderboard is saved as jupyter notebook.
+1. The jupyter notebook is executed and a modified pipeline model is  generated within the notebook.
+1. Pipeline model is deployed in Watson Machine Learning using WML APIs.
+
+# Included components
```

Comment:
Should not use a h1 (single #) except for the main title.
Also I don't usually do Included components or Feature tech anymore following the latest template (which has been around a while.  The components/featured go on the web page instead.
BUT... I know others still do this and I guess you can keep it if you prefer it.

Reply:
done. Will move out the included components to the main page and remove it from here.